scripts/install_pecan.sh

```diff
@@ -0,0 +1,739 @@
+    fi
+    if [ ! -e /etc/apt/sources.list.d/pgdg.list ]; then
+      sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt `lsb_release -s -c`-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+      wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+    fi
+    sudo apt-get -qq -y update
+    ;;
+esac
+
+echo "######################################################################"
+echo "INSTALLING PACKAGES"
+echo "######################################################################"
+case "$OS_VERSION" in
+  RH_*)
+    sudo yum install -y git R gfortran openssl-devel
+    sudo yum install -y openmpi openmpi-devel netcdf netcdf-openmpi netcdf-devel netcdf-openmpi-devel netcdf-fortran-devel netcdf-fortran-openmpi
+    sudo ln -s /usr/lib64/openmpi/bin/mpicc /usr/bin/mpicc
+    sudo ln -s /usr/lib64/openmpi/bin/mpif90 /usr/bin/mpif90
+    # for ED
+    sudo yum install -y hdf5-openmpi-devel
+    # for LPJ-GUESS
+    sudo yum install -y cmake
+    # for DALEC
+    sudo yum install -y gsl-devel liblas-devel lapack-devel
+    # for PEcAn
+    sudo yum install -y ed libpng-devel libpng12-devel libjpeg-turbo-devel jags4 jags4-devel python-devel udunits2-devel gdal-devel proj-devel proj-devel proj-nad proj-epsg libxml2-devel udunits2-devel gmp-devel
+    # for PostgreSQL
+    sudo yum install -y postgresql95-server postgresql95-devel postgis2_95
+    # web gui
+    sudo yum install -y httpd php php-pgsql php-xml
+    ;;
+  Ubuntu)
+    sudo apt-get -y install build-essential gfortran git r-base-core jags liblapack-dev libnetcdf-dev netcdf-bin bc libcurl4-gnutls-dev curl udunits-bin libudunits2-dev libgmp-dev python-dev libgdal1-dev libproj-dev expect
+    sudo apt-get -y install openmpi-bin libopenmpi-dev
+    sudo apt-get -y install libgsl0-dev
+    # for maeswrap
+    sudo apt-get -y install r-cran-rgl
+    # for R doc
+    sudo apt-get -y install texinfo texlive-latex-base texlive-latex-extra texlive-fonts-recommended
+    # ruby
+    sudo apt-get -y install ruby2.1 ruby2.1-dev
+    # for LPJ-GUESS
+    sudo apt-get -y install cmake
+    # for PostgreSQL
+    sudo apt-get -y install libdbd-pgsql postgresql-9.5 postgresql-client-9.5 libpq-dev postgresql-9.5-postgis-2.2 postgresql-9.5-postgis-scripts
+    sudo apt-get -y install php5-pgsql
+    # for web gui
+    sudo apt-get -y install apache2 libapache2-mod-php5 php5 libapache2-mod-passenger
+    ;;
+esac
+
+echo "######################################################################"
+echo "POSTGRES"
+echo "######################################################################"
+#    ADD export PATH=${PATH}:/usr/pgsql-9.5/bin
+#    ADD exclude=postgresql* to /etc/yum.repos.d/CentOS-Base.repo or /etc/yum/pluginconf.d/rhnplugin.conf 
+#    SEE https://wiki.postgresql.org/wiki/YUM_Installation#Configure_your_YUM_repository
+case "$OS_VERSION" in
+  RH_5)
+    echo "No PostgreSQL configuration (yet) for RedHat 5"
+    exit 1
```

Comment:
how about adding 'but the database can be run on a separate server from PEcAn; (see reference)'

Then either exit or 'continuing to install PEcAn without database' 

That said, I'm not sure how important it is to support RH 5